README.rst

```diff
@@ -11,3 +11,5 @@ PyMI
 Windows Management Infrastructure API for Python.
 
 For more details read the project's `README.rst <PyMI/README.rst>`_.
+
+rerun
```

Comment:
I think this was added by mistake.

Reply:
yeap, will remove.

Reply:
I've just realized that Python 3.9 was released as well in the meantime. We should add that to the list as well.